tracers.py

```diff
@@ -338,11 +340,11 @@ def get_nbins(self) :
         if ((self.tracer_type=="gal_clustering") or (self.tracer_type=="intensity_mapping") or
             (self.tracer_type=="gal_shear")) :
             data=np.loadtxt(self.bins_file,unpack=True)
-            if len(data)>5 :
-                self.lmax_bins=np.atleast_1d(data[5])
+            if len(data)>4 :
```

Comment:
I've implemented photo-z bias in this branch now, and that's why it had an extra column (see line 72 in the original file). The LSST files should be updated to reflect this change, but I haven't done that yet.